src/Components/Footer.js

```diff
@@ -15,7 +15,8 @@ class Footer extends Component {
            </ul> */}
 
            <ul className="copyright">
-              <li>&copy; Copyright 2020 GLUG MVIT</li>
+              <li>&copy; Copyright 2020 <a href="https://blog.glugmvit.com/" target="_blank" >GLUG MVIT</a> </li>
```

Comment:
Add `rel="noopener noreferrer"` to the anchor tag, using `target="_blank"` without those is a serious security vulnerabilty.
Read more here at  https://mathiasbynens.github.io/rel-noopener/

This is what is causing the builds to fail

Reply:
Thank you for letting me know.